src/containers/technology/TechnologySection.js

```diff
@@ -18,20 +18,33 @@ import * as Constants from './Constants';
  */
 
 const Content = styled.div`
-  margin-bottom: 150px;
-  margin-top: 150px;
+  margin: 80px 0 100px 0;
+  width: 100%;
+
+  @media only screen and (min-width: 768px) {
+    margin: 150px 0;
+  }
 `;
 
 const SectionHeaderWrapper = styled.div`
-  width: 500px;
+  width: 100%;
+
+  @media only screen and (min-width: 768px) {
+    width: 50%;
+  }
 `;
 
 const TechTilesWrapper = styled.div`
   display: flex;
   flex: 1 0 auto;
   flex-direction: row;
   flex-wrap: wrap;
-  justify-content: space-between;
+  justify-content: center;
```

Comment:
This doesn't look right when the window width is between ~660px & 768px. When the tiles are two per row, they should still maintain 30px margin between them.

![screen shot 2018-05-14 at 12 27 26 pm](https://user-images.githubusercontent.com/440299/40018805-32691244-5772-11e8-9a08-5bf90089c4ee.png)